oarepo_ui/resources/catalog.py

```diff
@@ -20,9 +25,95 @@
 class OarepoCatalog(Catalog):
     singleton_check = None
 
-    def __init__(self):
-        super().__init__()
-        self.jinja_env.undefined = jinja2.Undefined
+    def __init__(
```

Comment:
Do we have to overwrite the constructor? It seems that the only line modified is 51 and that is done to pass the current_app.

The app is used on 101: 
`if name in self.jinja_env.app.template_context_processors`

can it be changed to `if name in current_app.template_context_processors` ?

Also, we need tests for this functionality (register test blueprint with context processor and make sure it gets called)